tests/OverworldDiscordBot.py

```diff
@@ -0,0 +1,22 @@
+# Needs a discord bot account and a server with all the mark ids as emojis
+
+import asyncio
+import io
+import json
+import sys
+from functools import lru_cache
+from threading import Thread
+from time import sleep
+
+import discord
+import pokebase as pb
+from discord.ext import commands
+from discord.utils import get
+from PIL import Image
+
+sys.path.append('../')
+from discordbot import OverworldDiscordBot # Channels
+
+bot = OverworldDiscordBot()
+bot.configure(json.load(open("../config.json")))
+bot.run("ADD-TOKEN-HERE")
```

Comment:
having token be in config/a secure config to avoid needing to actually edit script is helpful, another temporary solution could be using an ``input()``?

Reply:
Is there a convention for something like config.json being excluded in a gitignore file, and the config.json is stored in something like a config.json.template file? So the user basically copies it, fills in config, and then it never gets checked in?

Reply:
including config.json as config.json.template and then gitignoring config.json would work, the user would have to copy config.json.template, remove .template, and then edit it for it to be used though. an instruction to do this would need to be added to readme

Reply:
Added some instructions to the readme, though they'll change before too long as this becomes even more usable.